common/djangoapps/student/models.py

```diff
@@ -1377,7 +1377,7 @@ def refundable(self):
 
         # If it is after the refundable cutoff date they should not be refunded.
         refund_cutoff_date = self.refund_cutoff_date()
-        if refund_cutoff_date and datetime.now() > refund_cutoff_date:
+        if refund_cutoff_date and datetime.utcnow() > refund_cutoff_date:
```

Comment:
I think this should be `datetime.now(pytz.UTC)`. `datetime.utcnow` returns a naive datetime object which can't be compared with timezone-aware datetimes (such as the return value of `self.refund_cutoff_date`.